css/style.css

```diff
@@ -1,11 +1,270 @@
-.section-category .img:hover {
+:root {
+  --light-grey: rgb(238, 238, 238);
+  --dark-black: hsl(180, 9%, 9%);
+}
+
+main.container {
+  margin-top: 20vh;
+  display: flex;
+  width: 100%;
+  height: auto;
+}
+button.menu {
+  display: none;
+}
+.search-icon {
+  margin-right: 0;
+  margin-left: auto;
+  visibility: hidden;
+}
+nav ul.left,
+nav ul.right {
+  flex-wrap: nowrap;
+}
+@media (max-width: 535px) {
+  header nav ul.left .input-box {
+    display: none;
```

Comment:
fix: This is very high specificity, you should reduce it for sure
also use @media only screen and max-width

Reply:
Done fixing